data/sles4sap/qe_sap_deployment/mr_test_ec2_product.yaml

```diff
@@ -0,0 +1,34 @@
+provider: "ec2"
```

Comment:
As it does not come from a setting here, I'd propose to use the `qe-sap-deployment` style name --> **aws**